params/config.go

```diff
@@ -380,6 +384,11 @@ func (c *ChainConfig) IsConstantinople(num *big.Int) bool {
 	return isForked(c.ConstantinopleBlock, num)
 }
 
+// IsEip2384 returns whether num is equal or greater than the EIP 2384 block
```

Comment:
Please copy paste the doc form a fork from above to have it consistent across all.